src/admin-views/tribe-options-display.php

```diff
@@ -123,6 +123,16 @@
 	)
 );
 
+$tribe_enable_views_tooltip = __( 'You must select at least one view.', 'the-events-calendar' );
```

Comment:
let's esc_html here as it'll be part of the tooltip. We don't want any translation to break this